components/style/bloom.rs

```diff
@@ -100,8 +100,8 @@ where
     E: TElement,
     F: FnMut(u32),
 {
-    f(element.local_name().get_hash());
-    f(element.namespace().get_hash());
+    f(element.local_name_hash());
+    f(element.namespace_hash());
```

Comment:
Same, this doesn't need to use `selectors::Element`, and `selectors` doesn't need `Element::namespace_hash`.